lib/routes/hanime1/previews.ts

```diff
@@ -0,0 +1,86 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import puppeteer from 'puppeteer';
```

Comment:
Improper use of puppeteer. Please have a look at the [Contribution Guide](https://docs.rsshub.app/joinus/new-rss/start-code).